src/EPR.Accreditation.API/Profiles/AccreditationProfile.cs

```diff
@@ -71,7 +72,46 @@ public AccreditationProfile()
             CreateMap<Data.MaterialReprocessorDetails, DTO.MaterialReprocessorDetails>()
                 .MapOnlyNonDefault()
                 .ReverseMap()
-                .MapOnlyNonDefault();
```

Comment:
I'm still passing this, but I've never been a fan of complicated mappings being done in the profile.